VimeoUpload/Upload/Descriptor System/Cameo/CAMUploadDescriptor.swift

```diff
@@ -12,15 +12,15 @@ import VimeoNetworking
 
 public class CAMUploadDescriptor: ProgressDescriptor, VideoDescriptor
 {
-    let videoUrl: NSURL
+    let videoUrl: URL
     let videoSettings: VideoSettings?
-    let thumbnailUrl: NSURL?
+    let thumbnailUrl: URL?
     
     public private(set) var uploadTicket: VIMUploadTicket?
-    private(set) var pictureTicket: VIMThumbnailUploadTicket?
-    private(set) var picture: VIMPicture?
+    private var pictureTicket: VIMThumbnailUploadTicket?
```

Comment:
These were previously private set and internal get, why are we making the gets private now?